timemachine/exchange/exchange_mover.py

```diff
@@ -125,7 +128,7 @@ def get_initial_state(water_pdb, mol, ff, seed, nb_cutoff):
     return initial_state, num_water_mols, solvent_topology
 
 
-class ExchangeMover:
+class ExchangeMove(moves.MonteCarloMove):
```

Comment:
Nice -- can now remove unused `a_count`, `t_count` fields.

(should be handled by `n_accepted`, `n_proposed` counters inherited from https://github.com/proteneer/timemachine/blob/68040c238553a210164cdb1dc01d0cb71cd394a1/timemachine/md/moves.py#L20-L22 )

Reply:
resolved in https://github.com/proteneer/timemachine/pull/1089/commits/8f127e1a5d185e55f2c1da99743876d1bf74a354